gallery_dl/extractor/instagram.py

```diff
@@ -37,7 +37,9 @@ def items(self):
             data.update(metadata)
             yield Message.Directory, data
 
-            if data['typename'] == 'GraphImage':
+            if data['typename'] == 'GraphImage' or \
```

Comment:
```suggestion
            if data['typename'] in ('GraphImage', 'GraphStoryImage', 'GraphStoryVideo'):
```

Reply:
Thanks, I have adjusted it as suggested!